src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -3055,6 +3055,115 @@ class LedgerRPC_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testLedgerEntryPermissionedDomain()
+    {
+        testcase("ledger_entry PermissionedDomain");
+
+        using namespace test::jtx;
+
+        Env env(*this, supported_amendments() | featurePermissionedDomains);
+        Account const issuer{"issuer"};
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+
+        env.fund(XRP(5000), issuer, alice, bob);
+        env.close();
+
+        auto const seq = env.seq(alice);
+        env(pd::setTx(alice, {{alice, "first credential"}}));
+        env.close();
+        auto const objects = pd::getObjects(alice, env);
+        BEAST_EXPECT(objects.size() == 1);
+        if (objects.empty())
+            return;
+        // env(pd::deleteTx(alice, domain));
+
+        {
+            // Succeed
+            Json::Value params;
+            params[jss::ledger_index] = jss::validated;
+            params[jss::permissioned_domain][jss::account] = alice.human();
+            params[jss::permissioned_domain][jss::seq] = seq;
+            auto jv = env.rpc("json", "ledger_entry", to_string(params));
```

Comment:
I'm not able to figure out the correct format of the `ledger_entry` command. I'm confident that the `sequence` and `account` values are correct in the below input. Am I mis-spelling permissioned_domain value?

```
Response(status=<ResponseStatus.ERROR: 'error'>, result={'api_version': 2, 'error': 'malformedRequest', 'id': 'RequestMethod.LEDGER_ENTRY_94839', 'ledger_current_index': 234, 'request': {'api_version': 2, 'binary': False, 'command': 'ledger_entry', 'id': 'RequestMethod.LEDGER_ENTRY_94839', 'permissioned_domain': {'account': 'ra5dMgd2j5DhKpW2h9beFsTsq7EKfDs4JJ', 'sequence': 220}}, 'status': 'error', 'type': 'response', 'validated': False}, id='RequestMethod.LEDGER_ENTRY_94839', type=<ResponseType.RESPONSE: 'response'>)
```

Reply:
@ckeshava you seem to forgot the `"params":[{...}]` array. Here example POST payloads taken from the above unit tests:

```
{"id":1,"method":"ledger_entry","params":[{"api_version":1,"ledger_index":"validated","method":"ledger_entry","permissioned_domain":{"account":"rG1QQv2nh2gr7RCZ1P8YYcBUKCCN633jCn","seq":4}}]}


{"id":1,"method":"ledger_entry","params":[{"api_version":1,"ledger_index":"validated","method":"ledger_entry","permissioned_domain":"272166035B5DFAD5EF4CFA7D782A2C1C207C7670F993DDCA50B8B184ACC3A926"}]}
```

If you want to peek into RPC test payloads, here is a very simple modification you can make in the sources yourself:

```
diff --git a/src/xrpld/net/detail/RPCCall.cpp b/src/xrpld/net/detail/RPCCall.cpp
index b92f4b1a20..f22822a852 100644
--- a/src/xrpld/net/detail/RPCCall.cpp
+++ b/src/xrpld/net/detail/RPCCall.cpp
@@ -85,6 +85,8 @@ createHTTPPost(

     s << "\r\n" << strMsg;

+    std::cerr << strMsg << "\n\n";
+
     return s.str();
 }
```

Reply:
Alright, never mind the previous message. I was using the field name `sequence` instead of `seq` in the ledger_entry request, my bad.